icicle/tests/test_field_api.cpp

```diff
@@ -907,106 +907,46 @@ TYPED_TEST(FieldApiTest, ntt)
 
 // define program
 using MlePoly = Symbol<scalar_t>;
-MlePoly combine_func(const std::vector<MlePoly>& inputs)
+
+void lambda_multi_result(std::vector<MlePoly>& vars)
 {
-  const MlePoly& A = inputs[0];
-  const MlePoly& B = inputs[1];
-  const MlePoly& C = inputs[2];
-  const MlePoly& EQ = inputs[3];
-  return (EQ * (A * B - C));
+  const MlePoly& A = vars[0];
+  const MlePoly& B = vars[1];
+  const MlePoly& C = vars[2];
+  const MlePoly& EQ = vars[3];
+  vars[4] = EQ * (A * B - C) + scalar_t::from(9);
+  vars[5] = A * B - !C;
```

Comment:
!C is field element inverse? if yes this is maybe a little confusing. Maybe use C.inverse() instead?

Reply:
ok. will be done in next PR